canonical-web-and-design/vanilla-framework
# Remove label component

## Done

- Remove all docs and examples of the deprecated `p-label` pattern
- Replace all instances on the site with chip pattern
- Add to what's new table
- Add to the upgrade guide
- Added text colour to chips 

Fixes

#4140 


## QA

- Open [demo](https://vanilla-framework-4154.demos.haus)
- Check the label component no longer exists in the docs/examples
- Review updated documentation:
  - Check what's new table as these labels have all been updated to use the chip
  - Check the 3.0 upgrade guide as a label section was added

### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
  - if CSS class names are not changed it can be bugfix relesase (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical-web-and-design/vanilla-framework/wiki/Release-process#pre-release-tasks)
